api/eid_oidc_provider/views.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import REDIRECT_FIELD_NAME
+from django.contrib.sites.shortcuts import get_current_site
+from django.http import HttpResponseRedirect, HttpResponseBadRequest
+from django.template.response import TemplateResponse
+from django.utils.http import is_safe_url
+from django.contrib.auth import login as auth_login
+from django.contrib.auth.models import User
+from django.shortcuts import resolve_url
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+from .utils.user import username_exists_in_db
+from eid_service.openid_interface import *
+
+
+def open_id_auth(request):
+
+    redirect_to='/api/eIdService/init'
+    params = None
+    if request.method == "GET":
+        params = request.GET
+    elif request.method == "POST":
+        params = request.POST
+
+    return HttpResponseRedirect(redirect_to + '?' + params.urlencode())
```

Comment:
preferably use django's [built-in method to redirect to other views](https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#django.urls.reverse)